stablehlo/dialect/StablehloOps.cpp

```diff
@@ -4686,7 +4697,8 @@ LogicalResult IfOp::inferReturnTypeComponents(
       adaptor.true_branch().front().getTerminator()->getOperandTypes();
   auto falseBranchRetureTypes =
```

Comment:
typo "return" ? for both the true and false branch variables

Reply:
Yeah, I can't find it in my code now, maybe fixed earlier :-)